aspnetcore/release-notes/aspnetcore-9/includes/newFeature1.md

```diff
@@ -0,0 +1 @@
+To prevent merge conflicts in ~/aspnetcore-9.md, new features should be documented in this include folder. Content in the include folder is not published unless explictily referenced in a topic.
```

Comment:
Should this be a comment?

Reply:
@danroth27  this can't be displayed because it's in the include folder. To be published, You would have to add a link to another file via:
```
[!INCLUDE[](~/release-notes/aspnetcore-9.0.md)]
```